apps/next/tamagui.config.ts

```diff
@@ -1,4 +1,7 @@
-import { tamaguiConfig } from '@react-native-templates/app'
```

Comment:
to fix the eg-preview and also make it independent of React native Templates, I removed import of Tamagui configuration since it is not really relevant. 

That way, we don't have to set up Expo, transpile React Native or alias packages, as there won't be any React Native dependencies pulled into the build by accident.

Reply:
ok seems reasonable